pkg/actuators/machine/instaces_test.go

```diff
@@ -91,7 +91,7 @@ func TestBuildEC2Filters(t *testing.T) {
 }
 
 func TestRemoveStoppedMachine(t *testing.T) {
-	machine, _, err := stubMachine()
+	machine, err := stubMachine()
```

Comment:
Can the test legitimately continue in the face of an error now? In the next test function we call `Fatal()` if there's an error - presumably this should be the case here too.